llvm/lib/Target/ARM/ARMTargetTransformInfo.cpp

```diff
@@ -2592,11 +2592,23 @@ void ARMTTIImpl::getUnrollingPreferences(Loop *L, ScalarEvolution &SE,
       return;
   }
 
+  bool Runtime = true;
+  if (ST->hasLOB()) {
```

Comment:
Can you add a comment explaining the reason?

Reply:
Done.